test/feature/test_siftdesc.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import kornia.testing as utils  # test utils
+
+from torch.testing import assert_allclose
+from torch.autograd import gradcheck
+from kornia.feature.siftdesc import *
+
+
+@pytest.mark.parametrize("ksize", [5, 13, 25])
+def test_get_sift_pooling_kernel(ksize):
+    kernel = get_sift_pooling_kernel(ksize)
+    assert kernel.shape == (ksize, ksize)
+
+
+@pytest.mark.parametrize("ps,n_bins,ksize,stride,pad",
+                         [(41, 3, 20, 13, 5),
+                          (32, 4, 12, 8, 3)])
+def test_get_sift_bin_ksize_stride_pad(ps,
+                                       n_bins,
+                                       ksize,
+                                       stride,
+                                       pad):
+    out = get_sift_bin_ksize_stride_pad(ps, n_bins)
+    assert out == (ksize, stride, pad)
+
+
+class TestSIFTDescriptor:
```

Comment:
what happens with patches with more than one channel ?

Reply:
Crash and it is intended: original SIFT is for single channels. Stuff like CSIFT is a less standard. 
Although I can add the flag to just apply SIFT to all channels, so the output would be multichannel as well. What do you think?

Reply:
looks good, go for it